test/navigation/navigation.cpp

```diff
@@ -5,24 +5,39 @@
 
 namespace hyped::test {
 
+bool is_float_eq(float a, float b)
+{
+  return ((a - b) < core::kEpsilon) && ((b - a) < core::kEpsilon);
+}
+
+void checkTrajectoriesEquality(const core::Trajectory &trajectory1,
+                               const core::Trajectory &trajectory2,
+                               bool equality)
+{
+  const bool comparision = is_float_eq(trajectory1.acceleration, trajectory2.acceleration)
+                           && is_float_eq(trajectory1.velocity, trajectory2.velocity)
+                           && is_float_eq(trajectory1.displacement, trajectory2.displacement);
+  ASSERT_EQ(equality, comparision);
```

Comment:
No equivalent `ASSERT_FLOAT_NE` and we can't just have three `ASSERT_NE` calls because if we have 2 trajectories {1,2,3} and {1,3,4} then it wouldn't work because `ASSERT_NE(1, 1)` would fail the test.

This was the easiest way i could think of during the meeting